module/platform/webots/Webots/src/Webots.cpp

```diff
@@ -108,6 +108,52 @@ namespace module::platform::webots {
         throw std::runtime_error("Unable to translate unknown NUgus.proto sensor name: " + name);
     }
 
+    ActuatorRequests make_acutator_request(const ServoTargets& commands, const DarwinSensors& sensors) {
```

Comment:
Should we make this return a ref?

Reply:
But the Actuator request that we make is on the stack, so if we return a ref it will be garbage.

Reply:
wouldn't it extend the lifetime???